tensorflow/compiler/xla/service/fusion_node_indexing_evaluation.cc

```diff
@@ -96,16 +96,7 @@ void FusionNodeIndexingEvaluation::UpdateIndexUsageCount(
     const HloInstruction* instruction) {
   int64 total = 0;
   for (const auto* user : indexing_users_[instruction]) {
-    int64 weight = 1;
```

Comment:
Can you please also remove the special case in tensorflow/compiler/xla/service/llvm_ir/fused_ir_emitter.cc ?
This is essentially the same code and still used for the FusionMerger pass.

Reply:
I was hesitant to do this as I won't test that change. But as you asked it, I included it.